packages/react/select/src/Select.tsx

```diff
@@ -272,6 +274,30 @@ const SelectValue = React.forwardRef<SelectValueElement, SelectValueProps>(
 
 SelectValue.displayName = VALUE_NAME;
 
+/* -------------------------------------------------------------------------------------------------
+ * SelectPlaceholder
+ * -----------------------------------------------------------------------------------------------*/
+
+const PLACEHOLDER_NAME = 'SelectPlaceholder';
+
+interface SelectPlaceholderProps {
+  children?: React.ReactNode;
+}
+
+const SelectPlaceholder: React.FC<SelectPlaceholderProps> = (
+  props: ScopedProps<SelectPlaceholderProps>
+) => {
+  const { __scopeSelect, children } = props;
+  const context = useSelectContext(PLACEHOLDER_NAME, __scopeSelect);
+
+  /* Portal the placeholder into the trigger value node when there is no value */
+  return context.value === undefined && context.valueNode && !context.valueNodeHasChildren
+    ? ReactDOM.createPortal(children, context.valueNode)
```

Comment:
given this doesn't render anything except the children, i'm curious what the thinking is behind making this a part instead of a prop on root?

Reply:
That's a good question. It started as a part originally. That said, I still feel like it's better as a component for a few reasons:
- "rendering" it inside the `Trigger` feels good seeing as that's where you want it
- It can be more than a string, and I feel like anything but a simple string reads pretty awkward in JSX as a prop

Arguments against:
- usually parts render an element
- it'll probably end up as a prop in people's abstractions anyway

Curious about other pros/cons to help make a decision!

Reply:
ah yeah, good points. 

i guess the prop could be on the `Value` part regarding your first point (to colocate it better), but yeah, if people want more than a string i agree with the "JSX in a prop" niggle. 

i wonder tho if most of the time it would be a string and this is potentially overheard for a stylistic preference that's not the 90% use-case?

Reply:
> i guess the prop could be on the Value part regarding your first point (to colocate it better)

That could be nice potentially

> but yeah, if people want more than a string i agree with the "JSX in a prop" niggle.

haha yeah, I hate to see JSX in props 😀, but again likely that's what it will become in all abstractions of the pattern… so? 🤷‍♂️ 

> i wonder tho if most of the time it would be a string and this is potentially overheard for a stylistic preference that's not the 90% use-case?

Yeah that's a good point too.

> only other point i can think of is SSR, prop could render placeholder from server.

How does a prop vs. part+`children` prop changes things here?

cc @andy-hook, would love to get your input here too so we can make a call.

Reply:
We are all leaning towards adding placeholder prop to Value part now, so we'll do that.